packages/angular_devkit/core/BUILD.bazel

```diff
@@ -16,6 +17,17 @@ licenses(["notice"])  # MIT License
 
 # @angular-devkit/core
 
+copy_to_bin(
```

Comment:
NB: this copy_to_bin is necessary because the linker is linking to bazel-out so any source files (such as `package.json`) that are required in the package need to be pushed to bazel-out with copy_to_bin. Not ideal but outside of the sandbox there is no better approach AFAIK. Side-effect is that this works on Windows which does not have a sandbox or runfiles.

Reply:
Without this, you'll get ```==================== Test output for //packages/angular_devkit/build_angular:build_angular_karma_test (shard 2 of 2):
internal/modules/cjs/loader.js:796
    throw err;
    ^

Error: Cannot find module '@angular-devkit/core'
Require stack:
- /private/var/tmp/_bazel_greg/dbb005a7eeb5fa8b89a9797588e36854/execroot/angular_cli/bazel-out/darwin-fastbuild/bin/packages/angular_devkit/build_angular/build_angular_karma_test.sh.runfiles/angular_cli/packages/angular_devkit/build_angular/src/karma/code-coverage_spec.js
- /private/var/tmp/_bazel_greg/dbb005a7eeb5fa8b89a9797588e36854/execroot/angular_cli/bazel-out/darwin-fastbuild/bin/packages/angular_devkit/build_angular/build_angular_karma_test.sh.runfiles/npm/node_modules/jasmine/lib/jasmine.js
```
because there is no package.json in the bazel-out folder of `@angular-devkit/core` which is linked